converters/toZigbee.js

```diff
@@ -6525,6 +6538,22 @@ const converters = {
             await entity.read('aqaraOpple', [0x0428], manufacturerOptions.xiaomi);
         },
     },
+    ZNCLBL01LM_hand_open: {
+        key: ['hand_open'],
+        convertSet: async (entity, key, value, meta) => {
+            await entity.write('aqaraOpple', {0x0401: {value: !value, type: 0x10}}, manufacturerOptions.xiaomi);
+        },
+        convertGet: async (entity, key, meta) => {
+            await entity.read('aqaraOpple', [0x0401], manufacturerOptions.xiaomi);
+        },
+    },
+    ZNCLBL01LM_reset_limits: {
+        key: ['reset_limits'],
+        convertSet: async (entity, key, value, meta) => {
+            const lookup = {'reset': 0};
+            await entity.write('aqaraOpple', {0x0402: {value: lookup[value], type: 0x10}}, manufacturerOptions.xiaomi);
+        },
+    },
```

Comment:
Was gonna add these via [`xiaomi_curtain_options`](https://github.com/Koenkk/zigbee-herdsman-converters/blob/44d420f1ea407560c8560f437bc8d2bac5d2e855/converters/toZigbee.js#L2492), but couldn't get the composite field needed to work properly

Seemingly same issue as https://discord.com/channels/556563650429583360/1050223315849056266/1050230498938589245